clang/lib/Sema/SemaExpr.cpp

```diff
@@ -6198,7 +6198,7 @@ struct ImmediateCallVisitor : public RecursiveASTVisitor<ImmediateCallVisitor> {
   bool VisitCallExpr(CallExpr *E) {
     if (const FunctionDecl *FD = E->getDirectCallee())
       HasImmediateCalls |= FD->isImmediateFunction();
-    return RecursiveASTVisitor<ImmediateCallVisitor>::VisitStmt(E);
+    return RecursiveASTVisitor<ImmediateCallVisitor>::VisitCallExpr(E);
```

Comment:
This is a noop, right? If so, I'd drop the changes here but make an NFC change for this and the one in `VisitSourceLocExpr()` at the same time.